minimum-constraints.txt

```diff
@@ -132,7 +134,9 @@ build==0.5.1
 Cython==0.29.22
 
 # Unit test (imports into testcases):
-packaging==19.0
+packaging==19.0; python_version <= '3.6'
+# Packaging 21.0 required by safety 2.2.0
+packaging==21.0; python_version >= '3.7'
```

Comment:
If you increase the minimum version here, you also need to increase it in test-requirements.txt.

Reply:
DONE